package.json

```diff
@@ -16,6 +16,9 @@
   },
   "homepage": "https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2#readme",
   "devDependencies": {
+    "eslint": "^8.5.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.25.3",
     "jest": "^26.6.3"
```

Comment:
Looks like you are missing the eslint plugin for jest here, try installing it with 
```
npm i eslint-plugin-jest
```
Afterward you should have something like this entry in your `package.json`
```
"eslint-plugin-jest": "^25.3.0"
```